OxfordBodleian/Juel-Jensen/BDLaethe38.xml

```diff
@@ -472,41 +471,38 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
-                                    <desc type="GuestText">Miracle of Jesus Christ</desc>
+                                    <desc type="GuestText"><title ref="LIT5992Miracle"/>
+                                        The note is hastily written. It narrates a miracle perfomed by Jesus Christ and mentions <persName ref="PRS12983GabraMa" role="patron"/>.
+                                        It might have been written in the same hand as that of <ref target="#a2"/>.
+                                    </desc>
                                     <locus from="78vb" to="80rb"/>
                                     <q xml:lang="gez">
                                         <seg part="I">
                                         <hi rend="rubric">ተአምረ፡ ዘገ<supplied reason="omitted">ብ</supplied>ረ፡</hi> እግዚእነ፡ ወአምላክነ፡ ኢየሱስ፡ ክርስቶ<supplied reason="omitted">ስ</supplied>
                                         በኃይለ፡ ሞቱ፡ <sic>ማኀየዊ፡</sic> <add place="above">ይምሐሮ፡ ለ</add>ገብሩ፡ ገብረ፡ ማርያም፡ ወሀሎ፡ ፩ድውይ፡ እንዘ፡ ይግ<add place="above">ዕ</add>ር፡ 
                                         ወየአውዩ፡ ፲<add place="above">፪</add>ዓመተ፡ ወዘረከቦ፡ በብዝኃ፡ ኃጢአቱ፡ ወተፈትነ፡ በእሳት፡ ከመ፡ ወርቅ፡ ወብሩር፡ መዓልተ፡ ወሌሊተ፡ </seg>
-                                    </q>
-                                    <note>
-                                        The note is hastily written. It narrates a miracle perfomed by Jesus Christ and mentions <persName ref="PRS12983GabraMa" role="patron"/>.
-                                        It might have been written in the same hand as that of <ref target="#a2"/>.
-                                    </note>
+                                    </q>                                  
                                 </item>
                                 <item xml:id="a4">
-                                    <desc type="GuestText" xml:id="LIT5921SalamLaki"/>
+                                    <desc type="GuestText"><title type="incomplete" ref="LIT5921SalamLaki"/> Part of the text is hardly readable due to faded ink. Spaces for rubrication have been left unfilled.</desc>
                                     <locus from="80va" to="80vb"/>
                                     <q xml:lang="gez">
                                         <seg part="I">
                                             ሰላም፡ ለኪ፡ ደብረ፡ ሊባኖስ፡ ግነት፡ ፄና፡ መዓዛኪ፡ ዕፍረት፡ ዘአዕረፈ፡ ብኪ፡ ተክለ፡ <choice><sic>ሃማኖተ፡</sic><corr>ሃይማኖት፡</corr></choice>
                                         </seg>
                                     </q>
-                                   <note>Part of the text is hardly readable due to faded ink. Spaces for rubrication have been left unfilled.</note>
                                 </item>
                                 
                                 <item xml:id="e1">
-                                    <desc>A white sticker is glued to the inner side of the front board. The name of <persName ref="PRS5782JuelJen"/> is printed on it
```

Comment:
I'm sorry, maybe I was confusing previously. I meant that desc should be used here, not note...